### DISCUSSION TITLE
表の偶数行目の中でインラインコードを見分けられるようにしてほしい

### DISCUSSION THREAD

**Discussion by Phroneris** [increments/qiita-discussions#246] [要望・アイデア・不具合]
Qiitaのマークダウンに用いられるスタイル（CSS）についての要望です。

次のように、表の中にインラインコードを書いた時、

```
| コード | 非コード | 混合 |
|:-:|:-:|:-:|
| `0OlI1` | 0OlI1 | 0 `O` `0` O |
| `0OlI1` | 0OlI1 | 0 `O` `0` O |
```

Qiitaではこのような色で表示されます。（スクショです）
![markdown_table_inline-code_discussions](https://user-images.githubusercontent.com/9891867/176891092-4d82be81-aadb-425b-87d0-f07663008b4f.png)

ちなみに、ここGitHubではこうです。（スクショではなく、マークダウンをそのままレンダリングさせているだけです）

| コード | 非コード | 混合 |
|:-:|:-:|:-:|
| `0OlI1` | 0OlI1 | 0 `O` `0` O |
| `0OlI1` | 0OlI1 | 0 `O` `0` O |

現状のQiitaでは、表の偶数行目の背景とインラインコードの背景が完全に同じ色で表示されるため、
偶数行目のセルの中でインラインコードの部分と通常のテキストの部分を見分けることが困難です。

両者を異なる背景色で表示してほしいです。

---

これは、Qiita公式記事『Markdown記法 チートシート』のコメント欄にて、[2020年8月に書いた要望](https://qiita.com/Qiita/items/c686397e4a0f4f11683d#comment-ab2ded8503e286ffc0aa)です。
今はより要望伝達にふさわしいこのような場が用意されているとのことで、少し調整しつつ改めてお伝えさせていただきました。

**Comment by tanaka-asami415** [comment]
@Phroneris 
Discussionsへの投稿ありがとうございます。ご不便をおかけして申し訳ありません🙏💦
こちらIssueも作成しているのですが、直近の開発優先度からまだ着手できておりません🙇
修正反映までご不便をおかけして申し訳ありませんが、お待ちいただけますと幸いです🙇💦

**Reply by tanaka-asami415** [reply]
@Phroneris 
お待たせいたしました！こちら、修正が完了いたしました！✨✨
[リリースノート](https://qiita.com/release-notes#2022-07-13)でも本日お知らせしております🙇
是非お試しいただけると嬉しいです✨

**Reply by Phroneris** [reply]
確認できました。
せっかくなのでまたスクショ：
![markdown_table_inline-code_discussions_fixed](https://user-images.githubusercontent.com/9891867/178715425-fe94e8e3-b611-4144-a316-734759d22efa.png)

迅速なご対応、誠にありがとうございました！

**Reply by tanaka-asami415** [reply]
@Phroneris 
こちらこそ、早速お試しくださりありがとうございます🙏✨
